### ISSUE TITLE
Eventually empty peers list (no peers, no connection) state

### ISSUE THREAD

**Issue by greenhat** [ergoplatform/ergo#1227]
I've experienced it twice already. Attached the log from running with jar built from v3.3.5 branch.

[ergo.log.gz](https://github.com/ergoplatform/ergo/files/5428639/ergo.log.gz)

```
OS: macOS 10.15.7
openjdk version "13.0.2" 2020-01-14
OpenJDK Runtime Environment AdoptOpenJDK (build 13.0.2+8)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 13.0.2+8, mixed mode, sharing)
```

During that time there were no network connectivity issues. After the restart, it's working fine.

**Comment by greenhat** [comment]
After 4-5 days of running fine (although peers count was quite low - 2-3) the node is having zero peers count. 
Output of `127.0.0.1:9053/peers/blacklisted` and `127.0.0.1:9053/peers/all` are empty.

**Comment by kushti** [comment]
Fixed in #1231
